Update Meeting Details API for a channel meeting (#4364)
MicrosoftDocs/msteams-docs: msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

The `Meeting Details` API enables your app to get static meeting metadata. The metadata provides data points that don't change dynamically.
The API is available through Bot Services.

> Now the API is supported in both private scheduled/recurring meeting and channel scheduled/recurring meeting with different a RSC permission respectively. Please check below Prerequisite section for more details. 

Reviewer: ```suggestion:
Currently, both private scheduled or recurring meetings and channel scheduled or recurring meeting supports API with different RSC permissions respectively. For more information, check the following prerequisite section.
```

#### Prerequisite

> To use the `Meeting Details` API, you must obtain RSC permissions. Depending on the scope of a meeting (private meeting or channel meeting), you need to use a different RSC permission. 

Reviewer: ```suggestion:
You need to use different RSC permission based on the scope of any meeting, such as private meeting or channel meeting, 
```
